opentracing_utils/span.py

```diff
@@ -5,12 +5,21 @@
 from opentracing.ext import tags as opentracing_tags
 
 
-def get_new_span(f, operation_name=None, inpsect_stack=True, ignore_parent_span=False, **kwargs):
+DEFAULT_SPAN_ARG_NAME = '__OPENTRACINGUTILS_SPAN'  # hmmm!
+
+
+def get_new_span(f, operation_name=None, inpsect_stack=True, ignore_parent_span=False, span_extractor=None, **kwargs):
     parent_span = None
-    span_arg_name = None
+    span_arg_name = DEFAULT_SPAN_ARG_NAME
 
     if not ignore_parent_span:
-        span_arg_name, parent_span = get_parent_span(inpsect_stack=inpsect_stack, **kwargs)
+        span_arg_name = DEFAULT_SPAN_ARG_NAME
```

Comment:
`span_arg_name` is already `DEFAULT_SPAN_ARG_NAME ` here

Reply:
Done.